libraries/AP_Math/AP_Math.cpp

```diff
@@ -9,8 +9,6 @@ bool is_equal(const FloatOne v_1, const FloatTwo v_2)
     return fabsf(v_1 - v_2) < std::numeric_limits<decltype(v_1 - v_2)>::epsilon();
 }
 
-template bool is_equal<int>(const int v_1, const int v_2);
-template bool is_equal<short>(const short v_1, const short v_2);
```

Comment:
What do we gain in removing then? The linker is smart enough to remove symbols not used. If there's a warnings in clang we can probably circumvent it by adding an attribute "maybe unused"